### PR TITLE
Loki docs deprecation

### PR DESCRIPTION
**Pull request by Teneroy**
**Description**

Changes proposed in this pull request:

- Added warnings concerning loki deprecation
- Added links to the loki installation tutrial
- Edited images related to loki usage in the system

**Related issue(s)**

#15893

### CODE REVIEW

File: docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md [hunk 1]
```diff
@@ -322,6 +326,8 @@ spec:
 
 ## Parameters
 
+> **NOTE:** Loki is [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) and is planned to be removed. If you want to install a custom Loki stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/loki).
```
Comment (rakesh-garimella) [inline docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md:329]: do we want to repeat this warning twice on same page.

**Comment by rakesh-garimella** [comment]
missing changes here we need to remove Loki link 
```
docs/01-overview/main-areas/observability/README.md
```

File: docs/05-technical-reference/00-architecture/obsv-02-architecture-logging.md [hunk 2]
```diff
@@ -2,9 +2,7 @@
 title: Logging Architecture
 ---
 
-## Architecture diagram
-
-![Logging architecture in Kyma](./assets/obsv-logging-architecture.drawio.svg)
```
Comment (chrkl) [inline docs/05-technical-reference/00-architecture/obsv-02-architecture-logging.md:7]: Delete the file as well

**Review by grego952 — APPROVED** [review]

**Pull request merged**